assets/src/utils/common.ts

```diff
@@ -141,9 +141,16 @@ export const parseBool = (val: any) => {
 
 /** returns a number if the string can be a number, else leaves it as a string */
 export const parseNumString = (item: string): string | number => {
+  const itemType = typeof item;
   if (!item?.length) return item;
-  // check if items are strings or numbers and converts if number
-  return !Number.isNaN(Number(item)) ? parseFloat(item) : item.trim();
+  if (!Number.isNaN(Number(item))) {
+    // check if items are strings or numbers and converts if number
+    return parseFloat(item);
+  } else if (itemType === 'string') {
+    //trim() only works on strings
+    return item.trim();
```

Comment:
item.trim(); caused the because it was throwing error when calling .trim() on an array object.